### ISSUE TITLE
How can I prepopulate my server address into my local instance of Jamstash?

### ISSUE THREAD

**Issue by adotm** [tsquillario/Jamstash#269]
I'd like to pre-populate my server address on my hosted Jamstash so I don't have to enter it in the setting page every time I try to log in.

Is this possible?

I tried to edit the settings.html file and add `value="address"` but I think the JS is overwriting it :(

**Comment by pR0Ps** [comment]
Yes, you have to make the change in the JS. Here's what I use to hardcode the server for my personal deployment (and also add a guest mode): https://github.com/tsquillario/Jamstash/commit/6d6e290939d988f824c33d30c62afb880f29dbb8 . It should be fairly straightforward to adapt that to your needs.
